resources/collect-dump-logs.sh

```diff
@@ -11,6 +11,11 @@ JVB_UID=`id -u jvb`
 RUNNING=""
 unset PID
 
+# for systemd we use different pid file
```

Comment:
why a different file

Reply:
Cause jvb user does not have the write to create files under /var/run. But specifying RuntimeDirectory it is created there before we create the pid file, and there the pid file is successfully created.

Reply:
But doesn't systemd writ the PID file?

Reply:
Nope. Having a specified pid file in the unit file is just for helping systemd the find the process or delete it ... but it is not creating pid files.

Reply:
Gotcha.